assemble/bin/accumulo

```diff
@@ -151,10 +151,23 @@ fi
 # Export the variables just in case they are not exported
 # This makes them available to java
 export JAVA_HOME HADOOP_PREFIX ZOOKEEPER_HOME LD_LIBRARY_PATH DYLD_LIBRARY_PATH
+
```

Comment:
I only tested using the other start/stop scripts, not running the accumulo script. If you start the processes using only the accumulo script, then I don't think you will pick up the NUMA options. It appears that in 1.8 the start-daemon.sh script is the preffered way for starting processes.